kolibri/core/content/utils/delete_content.py

```diff
@@ -0,0 +1,12 @@
+from kolibri.core.content.models import LocalFile
+
+import logging
+logger = logging.getLogger(__name__)
+
+def cleanup_unavailable_stored_files():
+    deleted_files = LocalFile.objects.delete_unavailable_stored_files()
+    freed_bytes = sum((f.file_size for f in deleted_files), 0)
+    logger.info(
+        "Cleaned up %i unused files, freeing %i bytes." % ( len(deleted_files), freed_bytes )
+    )
```

Comment:
Maybe return the freed_bytes from the function, and add a test that asserts that this returns a true value?